docs/USER_GUIDE.md

```diff
@@ -104,6 +104,15 @@ without any arguments:
 Since `krew` itself is a plugin also managed through `krew`, running the upgrade
 command may also upgrade your `krew` version.
 
+### Krew upgrade check
+
+When using krew, it will check if a new version of krew is available once a day
+by calling the GitHub API. If you want to opt out of this feature, you can set
+the `KREW_NO_UPGRADE_CHECK` environment variable. To permanently disable this,
+add the following to your `~/.bashrc`, `~/.bash_profile`, or `~/.zshrc`:
+
+    export KREW_NO_UPGRADE_CHECK=1
```

Comment:
An idea: since we don't know how it will affect users in the beginnning, we might make it opt-in to use at first.

Reply:
-1. we discussed at #480 that this isn't sending PII. It's basically almost like a git clone that we already do. I don't wish users to learn yet another thing. Let's keep the interaction surface minimal.

Reply:
I agree, sounds good. 

Btw, sorry for skipping the actual discussion and opening them here again.